File: C_Programming Assignment/Assignment_2/programAQ2.c
///////////////////////////////////////////////////////////
//
//  File name : Assigment2Question2.c
//  Descreption : Used to accept one number from user & print that number of * on screen
//  Author : Kartik Ganesh Jare
//  Date : 09/05/25
//  
///////////////////////////////////////////////////////////

#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;  
   
    while(iNo > 0)
    {
        printf("*\n");
        iNo --;
    }
}

int main()
{
    int iValue = 0;

    printf("Enter number :");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;

}

File: Java_Assignment/Assignment_1/programAQ2.java
///////////////////////////////////////////////////////////
//
//  File name : Assigment1Question2.java
//  Descreption : Used to print 5 times"Marvellous" on screen
//  Author : Kartik Ganesh Jare
//  Date : 31/05/25
//  
///////////////////////////////////////////////////////////

class Display
{
    public void Display()
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= 5; iCnt++)
        {
            System.out.println("Marvellous");
        }    
    }
}

class programAQ2
{
    public static void main(String A[])
    {
        Display dobj = new Display();
        dobj.Display();
    }
}